### ISSUE TITLE
plotting validation accuracy during training the model

### ISSUE THREAD

**Issue by mpc001** [espnet/espnet#197]
Hi,

The curve "acc.png" plotted in the stage "Network Training" presenting the DNN accuracy. We can also predict the sequence using beam search and get a CER report in the stage "Decoding". However, A high classification accuracy in DNN but a low ASR performance, such as CER obtained in decoding.

Many thanks for your help!

**Comment by sw005320** [comment]
I'm expecting that search parameters (penalty, maxlenratio, minlenratio) are not appropriately set (I'm also assuming that you're using attention-encoder-decoder without CTC from #186). You can tune these parameters to get some reasonable numbers.
Also, if you tell me the insertion, deletion, and substitution errors, I can tell you how we change the search parameters.

**Comment by mpc001** [comment]
Hi, @sw005320 ,

I list the detail information of validation set in ASR below. In this case, the frames we used is very close to the number of character we would predict.
 Corr                 Sub                Del                 Ins                Err               S.Err
52.3                30.3               17.3                13.8               61.5                99.0

Many thanks for your help!

**Comment by sw005320** [comment]
Thanks. The decoding parameters does not seem to be very strange.  

> the frames we used is very close to the number of character we would predict

Then, I think subsampling (we usually did to make the input length 1/4) seems to be a problem. I think you may think of tuning it. Subsampling is handled by `run.sh --subsample 1_2_2_1_1 --etype blstmp` (note that it is only handled by the blstmp architecture). If you change it as `1_1_1_1_1`, it will not perform subsampling.

**Comment by mpc001** [comment]
Hi, @sw005320,

Thanks for your kind reply. 

The above results are obtained after changing from 1_2_2_1_1 to 1_1_1_1_1. Is it weird that the CTC weight increases WER but not CER? The encoding part uses mtlalpha=0.5. The decoding part beam_size=20, penalty=0, maxlenratio=0.0 minlenratio=0.0.

![1](https://user-images.githubusercontent.com/28733498/41004363-867db99c-6911-11e8-976b-1d438864be79.png)

Many thanks for your help!

**Comment by sw005320** [comment]
Thanks for showing interesting results. Frankly, I did not observe such behaviors, as I did not investigate these parameters so much in terms of both CER and WER. If I come up with some reasons, I'll let you know.
